solutions/usaco-917.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: usaco-917
+source: USACO Bronze February 2019
+title: Measuring Traffic
+author: Brad Ma
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_traffic_bronze_feb19.html)
+
+## Implementation
+**Time Complexity** $\mathcal{O}(N)$
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class MeasuringTraffic {
+	static final int infinity = 999999999;
```

Comment:
would something like integer.max_value be better for this?

Reply:
great minds think alike 😆